CHANGELOG.md

```diff
@@ -1,3 +1,67 @@
+## v0.22.0 (2023-11-15)
+
+> Note: The `--topic` option is now deprecated in favor of a more specific options `--pubsub-topic` & `--content-topic`
+
+> Note: The `ext-multiaddr-only` CLI flag was introduced for cases in which the user wants to manually set their announced addresses
```

Comment:
Maybe better referring to it as a switch if its that.
```suggestion
> Note: The `--ext-multiaddr-only` CLI flag was introduced for cases in which the user wants to manually set their announced addresses
```

Reply:
Oh great catch! Thank you!